### ISSUE TITLE
Update CI for isort version 5. 

### ISSUE THREAD

**Issue by carltongibson** [django/channels#1462]
Pinned to <5 in travis.yml. 

Need to work out what the v5 changes were. Maybe just removing the `--recursive`? 

//cc @smithdc1 — I have a feeling you know the answer here.

**Comment by smithdc1** [comment]
tl;dr. Give it a few weeks before looking into making the move to v5.

Certainly need to remove the `--recursive`  option. This should get it to work but the formatting of the imports is very likely to change, and change again in coming weeks as there are a number of bugs. Version 5 has already had 5(!) bug fix releases since Saturday with still a dozen or open issues which have been raised in the last couple of days. 

So if no rush may be well be worth waiting a little while for it to settle down to avoid spending effort now for a bug fix release to require further changes. 

From my review at the weekend on Django itself, there were a number of issues (I couldn't even get it to apply consistently) and some of the changes were a little surprising.

**Comment by carltongibson** [comment]
Super. Thanks for update.
